src/hooks/useAnimationLoop/useAnimationLoop.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { renderHook } from '@testing-library/react';
+import { useAnimationLoop } from './useAnimationLoop.js';
+
+describe('useAnimationLoop', () => {
```

Comment:
This could use some additional tests.

- provide a callback, with a spy, to make sure it calls it
- see if it calls it twice by waiting some time (not sure if we can use jest fake timers here)
- pass a new callback function after the 1st render/invocation, and make sure it doesn't call the old one, but only the new one
- test pausing/unpausing the loop

Reply:
added test for every case @ThaNarie mentioned except this one:
_see if it calls it twice by waiting some time (not sure if we can use jest fake timers here)_

Reply:
We cannot test this using fake timers, real timers are not reliable enough to get consistent test results either. I've added the delta time to the storybook to visualise the animation loop timing.

![image](https://github.com/mediamonks/react-hooks/assets/5376407/3d5ef8bb-7a6d-4a9e-a39c-f2307062ecf5)